examples/seismic/acoustic/acoustic_example.py

```
@@ -1,5 +1,8 @@
 import numpy as np
-import pytest
+try:
+    import pytest
+except:
```

Bare except; should this be `ModuleNotFoundError`?